src/web_app_skeleton/docker/development.dockerfile

```diff
@@ -0,0 +1,36 @@
+FROM crystallang/crystal:1.1.1
```

Comment:
This needs to be updated, but later versions of the crystal dockerfile don't play well with my computer yet. If someone is willing to test this on an AMD64 architecture with this line updated to 1.3.2, I'll be happy to update it.